packages/harmony/src/storybook/components/ComponentRule.tsx

```diff
@@ -0,0 +1,54 @@
+import type { CSSProperties, ReactElement } from 'react'
+
+// TODO move and use ValidationCheck from completion-check
```

Comment:
The CompletionCheck component is in, did you want to use that here?

Reply:
yeah let me try that, and will reach out to you with what i find.